components/src/core/drawer/drawer-layout/drawer-layout.component.scss

```diff
@@ -8,6 +8,9 @@ $transition: 300ms cubic-bezier(0.4, 0, 0.2, 1);
 /* RTL */
 [dir='rtl'] .pxb-drawer-layout-content .mat-drawer-side {
     border-left: 0; // Border-left is provided by the Drawer component.
+    .pxb-drawer-layout-sidenav.pxb-drawer-layout-side-border {
+        box-shadow: 0 0 -4px 0 #424e54;
+    }
```

Comment:
should these colors come from the theme? Or at least the Colors package?

Reply:
Since the pxb-angular-comp library doesn't have a peerDependency on our colors package we can add an override for this color in our themes package.  This can be another instance of having a default color baked into our library, which we just override with our pxb & pxb-dark theme.